include/aikido/control/ros/util.hpp

```diff
@@ -0,0 +1,44 @@
+#ifndef AIKIDO_CONTROL_ROS_UTIL_HPP_
+#define AIKIDO_CONTROL_ROS_UTIL_HPP_
+#include <chrono>
+#include <mutex>
+#include <ros/ros.h>
+#include <ros/callback_queue.h>
+#include <actionlib/client/action_client.h>
+
+namespace aikido {
+namespace control {
+namespace ros {
+
+template <class ActionSpec, class TimeoutDuration, class PeriodDuration>
+bool waitForActionServer(
+  actionlib::ActionClient<ActionSpec>& actionClient,
+  ::ros::CallbackQueue& callbackQueue,
+  TimeoutDuration timeoutDuration = std::chrono::milliseconds{ 1000 },
+  PeriodDuration periodDuration = std::chrono::milliseconds{ 10 }
+)
```

Comment:
This function needs a docstring. Also, it should only be _declared_ in the `util.hpp` file. The implementation should be `#included` from a `detail/util-impl.hpp` file to match the convention we follow inn the rest of Aikido.